templates/download/risc-v/milk-v-titan.html

```diff
@@ -0,0 +1,32 @@
+<div tabindex="0" role="tabpanel" id="milk-v-titan-tab" aria-labelledby="milk-v-titan"
+  aria-hidden="true">
+  <div class="grid-row p-section--shallow">
+    <div class="grid-col-2 grid-col-medium-1">
+      <h3 class="u-hide--small p-heading--2">Milk-V Titan</h3>
+    </div>
+    <div class="grid-col-4 grid-col-medium-2 u-vertically-center u-align--center">
+      <div class="p-image-container is-highlighted">
+        {{ image(url="https://assets.ubuntu.com/v1/8748959d-milk_v_titan.png",
+          alt="",
+          width="1727",
+          height="1263",
+          hi_def=True,
+          loading="auto",
+          attrs={"class": "p-image-container__image"}) | safe
+        }}
+      </div>
+    </div>
+  </div>
+  <div class="grid-row">
+    <hr class="p-rule--muted" />
+    <div class="grid-col-2 grid-col-medium-1">
+      <h4 class="p-heading--5">Ubuntu 24.04 Server preinstalled image</h4>
+    </div>
+    <div class="grid-col-4 grid-col-medium-2">
+      <p>Please follow Milk-V’s instructions when installing the Milk-V Titan image.</p>
+      <div class="p-cta-block">
+        <a href="https://github.com/milkv-titan" class="p-button">Download Milk-V Titan image</a>
```

Comment:
This link goes to an [organization](https://github.com/milkv-titan) with no public repositories, will there be public data there as part of some launch?